tests/queries/0_stateless/00949_format.sql

```diff
@@ -226,3 +226,5 @@ select format('{}{}{}', materialize(toFixedString('a', 1)), toFixedString('b', 1
 select format('{}{}{}', materialize(toFixedString('a', 1)), toFixedString('b', 1), materialize(toFixedString('c', 1))) == 'abc' from system.numbers limit 5;
 select format('{}{}{}', materialize(toFixedString('a', 1)), materialize(toFixedString('b', 1)), toFixedString('c', 1)) == 'abc' from system.numbers limit 5;
 select format('{}{}{}', materialize(toFixedString('a', 1)), materialize(toFixedString('b', 1)), materialize(toFixedString('c', 1))) == 'abc' from system.numbers limit 5;
+
+SELECT isConstant(format('{}, world', 'Hello'));
```

Comment:
It's much better to add new test cases to new tests instead of modifying the existing ones.

The advantages are:
- you get a more granular report when only a single case has failed;
- the history of test runs is consistent;
- it is easier to attribute tests to particular changes in the code.

Reply:
Okay, these reasons are valid. Will do.